release/authorization/authorization.proto

```diff
@@ -0,0 +1,125 @@
+// Authorization for gnXi operations.
+//
+// Define a list of policies, where the most specific policy is applied
+// on the device authorizing the action.
+//
+// Users may be referenced by user name, or as a group with a group name.
+// Groups are defined and managed in the authorization policy itself.
+//
+// Paths may be referenced in whole or in complete parts, ie:
+//   /interfaces/interface[name=Ethernet1/2/3]/state/counters
+//   /interfaces/interface[name=*]/state/oper-status
+//   /network-instances/network-instance/tables/table[protocol=BGP][address-family=*]
+//
+// Paths are gnmi.Path protobufs.
+//
+// The most specific match is returned for a request, this means the longest
+// path and the most specific user match (user preferred over group).
+//
+// Additionally, the policy maintains a timestamp of creation
+// and version number from the underlying version control system.
+// The version/timestamp are available to requestors to verify which version
+// of policy is being applied at the time of the request.
+//
+syntax = "proto3";
+
+package authorization.proto;
+
+import "github.com/openconfig/gnmi/proto/gnmi/gnmi.proto";
+
+option go_package = "github.com/openconfig/public/release/authorization";
```

Comment:
I'll assume if this is going to fall under gNSI then this content will likely make its way to a separate repo (following suit w/ gnmi, gnoi, etc..) and thus go_package name altogether?

Reply:
probably, yes. Or, I don't really care where the package lands, someone in OC land can determine the best placement :) when they do we can update this path.